config/application.rb

```diff
@@ -22,5 +22,9 @@ class Application < Rails::Application
     config.govuk_notify_templates = config_for(
       :govuk_notify_templates, env: ENV.fetch('GOVUK_NOTIFY_ENV', 'development')
     ).symbolize_keys
+
+    # Prevents surrounding of erroneous inputs with <div class="field_with_errors">
+    # https://guides.rubyonrails.org/configuring.html#configuring-action-view
+    config.action_view.field_error_proc = proc { |html_tag| html_tag.html_safe }
```

Comment:
I had to add this otherwise the div added by Rails when there is an error messes up with the gov-uk styling

Reply:
Again, have a look at the `GovUkFormHelper` as I did manage to get GovUK styled error to work with radio buttons.

Reply:
But the existing component doesn't prepopulate the radio button with an existing value